src/main/java/ch/njol/skript/util/Utils.java

```diff
@@ -518,8 +512,8 @@ public static CompletableFuture<ByteArrayDataInput> sendPluginMessage(Player pla
 	final static Map<String, String> chat = new HashMap<>();
 	final static Map<String, String> englishChat = new HashMap<>();
 
-	public final static boolean HEX_SUPPORTED = Skript.isRunningMinecraft(1, 16);
-	public final static boolean COPY_SUPPORTED = Skript.isRunningMinecraft(1, 15);
+	public final static boolean HEX_SUPPORTED = true;
+	public final static boolean COPY_SUPPORTED = true;
```

Comment:
if these are statically true, couldn't they just be removed?

Reply:
I guess the only reason for keeping them might be to preserve compatibility if some addon was using them for its own checks, but I think it's unlikely to be the case.